admin_guide/managing_pods.adoc

```diff
@@ -219,32 +331,63 @@ endif::openshift-origin[]
     securityContext:
       privileged: true
     env:
-    - name: EGRESS_SOURCE <1>
+    - name: EGRESS_SOURCE <2>
       value: 192.168.12.99
-    - name: EGRESS_GATEWAY <2>
+    - name: EGRESS_GATEWAY <3>
       value: 192.168.12.1
-    - name: EGRESS_DESTINATION <3>
+    - name: EGRESS_DESTINATION <4>
       value: 203.0.113.25
   nodeSelector:
-    site: springfield-1 <4>
-----
-<1> IP address on the node subnet reserved by the cluster administrator for use by
-this pod.
-<2> Same value as the default gateway used by the node itself.
-<3>  Connections to the pod are redirected to 203.0.113.25, with a source IP
-address of 192.168.12.99
-<4> The pod will only be deployed to nodes with the label site *springfield-1*.
+    site: springfield-1 <5>
+----
+<1> The `pod.network.openshift.io/assign-macvlan annotation` creates a Macvlan
+network interface on the primary network interface, and then moves it into the
+pod's network name space before starting the *egress-router* container. Preserve the the quotation marks around `"true"`. Omitting them will result in
+errors.
+<2> An IP address from the physical network that the node itself is on and is
+reserved by the cluster administrator for use by this pod.
+<3> Same value as the default gateway used by the node itself.
+<4> The external server that to direct traffic to. Using this example, connections
+to the pod are redirected to 203.0.113.25, with a source IP address of
+192.168.12.99.
+<5> The pod will only be deployed to nodes with the label `springfield-1`.
```

Comment:
\`site=springfield-1\`, not just \`springfield-1\`. This line should be exactly the following text:

    <5> The pod will only be deployed to nodes with the label `site=springfield-1`.